api/v1/prefectworkpool_types.go

```diff
@@ -44,16 +45,31 @@ type PrefectWorkPoolSpec struct {
 	// Resources defines the CPU and memory resources for each worker in the Work Pool
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
 
-	// A list of environment variables to set on the Prefect Server
+	// A list of environment variables to set on the Prefect Worker
 	Settings []corev1.EnvVar `json:"settings,omitempty"`
 }
 
 type PrefectServerReference struct {
-	// Namespace is the namespace where the Prefect Server is running
+	// Namespace is the namespace where the in-cluster Prefect Server is running
```

Comment:
wonder if we make `Name` and `Namespace` optional now

Reply:
seems like we could at least make Namespace optional, will think on it some more